src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -217,18 +217,16 @@ public void init() {
      * @throws IllegalStateException when validation of default javadoc tokens fails
      */
     private void validateDefaultJavadocTokens() {
-        if (getRequiredJavadocTokens().length != 0) {
```

Comment:
to kill mutation you need to find javadoc check that has no required tokens and use it
or make test Check that has no required tokens and execute some validation in test.
We have several Checks that are defined in Test class as inner class, so just search in sources for example.

Reply:
Ok... was occupied for sometime, will start working on this now.